src/framework/theme/components/autocomplete/autocomplete.directive.ts

```diff
@@ -327,10 +327,8 @@ export class NbAutocompleteDirective<T> implements OnDestroy, AfterViewInit, Con
   }
 
   protected handleInputValueUpdate(value: T) {
-    if (value === undefined || value === null) {
-      return;
-    }
-    this.setHostInputValue(value);
+    const displayValue = value === undefined || value === null ? '' : value;
+    this.setHostInputValue(displayValue);
```

Comment:
What do you think about the following refactoring?
```suggestion
    this.setHostInputValue(value ?? '');
```